### PR TITLE
CATTY-278 Implement PenUpBrick

### PR DESCRIPTION
**Pull request by neel-makhecha**
- Implemented PenUpBrick, its instruction and serialization with tests.
- Implemented PenBrickCell

### Your checklist for this pull request
Please review the [contributing guidelines](https://github.com/Catrobat/Catty/blob/develop/README.md) and [wiki pages](https://github.com/Catrobat/Catroid/wiki/) of this repository.

- [ ] Include the name of the Jira ticket in the PR’s title
- [ ] Verify that the Jira ticket is in the status *Ready for Development*
- [ ] Include a summary of the changes plus the relevant context
- [ ] Choose the proper base branch (*develop*)
- [ ] Confirm that the changes follow the project’s coding guidelines
- [ ] Verify that the changes generate no compiler or linter warnings
- [ ] Perform a self-review of the changes
- [ ] Verify to commit no other files than the intentionally changed ones
- [ ] Include reasonable and readable tests verifying the added or changed behavior
- [ ] Confirm that new and existing unit tests pass locally
- [ ] Check that the commits’ message style matches the [project’s guideline](https://github.com/Catrobat/Catroid/wiki/Commit-Message-Guidelines)
- [ ] Stick to the project’s git workflow (rebase and squash your commits)
- [ ] Verify that your changes do not have any conflicts with the base branch
- [ ] After the PR, verify that all CI checks have passed
- [ ] Post a message in the *#catty* [Slack channel](https://catrobat.slack.com) and ask for a code reviewer

### CODE REVIEW

File: src/Catty/Parser&Serialializer/New Parser & Serializer/XMLHandler/Bricks/Pen/PenUpBrick+CBXMLHandler.swift [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see http://www.gnu.org/licenses/.
+ */
+
+extension PenUpBrick: CBXMLNodeProtocol {
+    static func parse(from xmlElement: GDataXMLElement!, with context: CBXMLParserContext!) -> Self! {
```
Comment (m-herold) [inline src/Catty/Parser&Serialializer/New Parser & Serializer/XMLHandler/Bricks/Pen/PenUpBrick+CBXMLHandler.swift:24]: Please remove the implicit force unwrapping (also for the _PenDownBrick_).

File: src/Catty/Parser&Serialializer/New Parser & Serializer/XMLHandler/Bricks/Pen/PenUpBrick+CBXMLHandler.swift [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+        CBXMLParserHelper.validate(xmlElement, forNumberOfChildNodes: 0)
+        return self.init()
+    }
+
+    func xmlElement(with context: CBXMLSerializerContext!) -> GDataXMLElement! {
```
Comment (m-herold) [inline src/Catty/Parser&Serialializer/New Parser & Serializer/XMLHandler/Bricks/Pen/PenUpBrick+CBXMLHandler.swift:29]: Please remove the implicit force unwrapping for the parameter and return an optional (also for the _PenDownBrick_).

**Comment by neel-makhecha** [comment]
Hi Michael, I have updated the PR. Please review.

**Comment by neel-makhecha** [comment]
Done!

**Pull request merged**